Description: Fix for sphinx7.1 with extlinks
Author: Nilesh Patra <nilesh@debian.org>
Last-Update: 2023-11-10
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -225,7 +225,7 @@
 # If false, no module index is generated.
 # latex_use_modindex = True
 
-extlinks = {"issue": ("https://github.com/pydata/bottleneck/issues/%s", "#")}
+extlinks = {"issue": ("https://github.com/pydata/bottleneck/issues/%s", "#%s")}
 intersphinx_mapping = {
     "numpy": ("https://docs.scipy.org/doc/numpy/", None),
     "numpydoc": ("https://numpydoc.readthedocs.io/en/latest/", None),
